lib/setup/setupInstall.js

```diff
@@ -1,8 +1,10 @@
+//@ts-check
+
 'use strict';
 
 function Install(options) {
     var fs       = require('fs');
-    var tools    = require(__dirname + '/../tools.js');
+    var tools    = require('../tools.js');

```

Comment:
AFAIK, `require` is always relative to the current file path, so these two lines should be equivalent.
The upside of the relative path is that VSCode detects the contents of `tools.js` and provides auto-completion and error checking.

Reply:
I remember you edited a similar change back to `__dirname` on another PR. Is there any reason we can't use relative paths?

Reply:
Ok. Just tested to change the CWD and start from other directory and it works.
I had with some node.js version that './something.js' was not in the current directory but with the WORKING Directory and __dirname works well.
But now seems to work.

Reply:
This is great. In the long run I want to extend the typings for a bunch of core classes. And this helps a ton because the typings will then be accessible in the importing classes aswell.